### PR TITLE
New js sdk reference docs

### PR DESCRIPTION
**Pull request by sublimezeal**
New per-function reference docs for the javascript SDK

### CODE REVIEW

File: docs/sdks/javascript-sdk/Reference/js-sdk-deleteCredential.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+2. Initialize the SDK
+```javascript
+const embedded = await Embedded.initialize();
+```  
+
+## Parameters
+| Parameter | Type |Description|
+|---|---|---|
+|id| string| Required. The ID of the passkey to be deleted. This should match the **id** property of a [Passkey](.\js-sdk-passkey-type.md) that is available on the device.|
```
Comment (Will5) [inline docs/sdks/javascript-sdk/Reference/js-sdk-deleteCredential.md:29]: I think you meant this to be `./js-sdk-passkey-type.md`

File: docs/sdks/javascript-sdk/Reference/js-sdk-getCredentials.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+2. Initialize the SDK
+```javascript
+const embedded = await Embedded.initialize();
+```  
+
+## Parameters
+none
+
+## Returns
+Upon success, returns a Promise that resolves to an array of [Passkey](.\js-sdk-passkey-type.md).
```
Comment (Will5) [inline docs/sdks/javascript-sdk/Reference/js-sdk-getCredentials.md:30]: I think you meant this to be `./js-sdk-passkey-type.md`

**Comment by turnipdabeets** [comment]
These look nice! I think you and @Will5 might want to sync up. Will's joining all the overviews into one file and maybe you can merge some of your work into there.

**Comment by ismailmustafa** [comment]
> These look nice! I think you and @Will5 might want to sync up. Will's joining all the overviews into one file and maybe you can merge some of your work into there.

@turnipdabeets do you mean the SDK Setup? The reference documentation that @sublimezeal is working on is different from that.

File: docs/sdks/javascript-sdk/Reference/js-sdk-bindcredential.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+## Examples
+### Example: Call **bindPasskey** after validating passkey binding URL
+```javascript
+if (await embedded.isBindPasskeyUrl(passkeyBindingLink)) {
+      let result = await embedded.bindPasskey(passkeyBindingLink);
+      //consume result and redirect the user
+}
+```
+
+### Example: Create identity and get passkey binding URL
```
Comment (turnipdabeets) [inline docs/sdks/javascript-sdk/Reference/js-sdk-bindcredential.md:82]: You might want to link to the #207 guide

**Review by ismailmustafa — APPROVED** [review]

**Pull request merged**